docs/esa/xmm_newton/xmm_newton.rst

```diff
@@ -85,79 +88,79 @@ This will download the EPIC postcard for the observation '0505720401' and it wil
 This function provides access to the XMM-Newton Science Archive database using the Table Access Protocol (TAP) and via the Astronomical Data
 Query Language (ADQL).
 
-.. code-block:: python
+.. doctest-remote-data::
 
   >>> from astroquery.esa.xmm_newton import XMMNewton
   >>>
-  >>> result = XMMNewton.query_xsa_tap("select top 10 * from v_public_observations", output_format='csv', output_file='results10.csv')
-  >>> print(result)
+  >>> result = XMMNewton.query_xsa_tap("select top 10 * from v_public_observations",
+  ...                                  output_format='csv', output_file='results10.csv')
+  >>> print(result)     # doctest: +IGNORE_OUTPUT
           bii             dec    duration ...      target      with_science
   -------------------- --------- -------- ... ---------------- ------------
-  4.1331178178373715  55.99944    32913 ...    XTE J0421+560         true
-  0.05069186370709437 -32.58167    17083 ...         HD159176         true
-  0.05069186370709437 -32.58167     9362 ...         HD159176         true
-  0.05069186370709437 -32.58167    10859 ...         HD159176         true
+    4.1331178178373715  55.99944    32913 ...    XTE J0421+560         true
+   0.05069186370709437 -32.58167    17083 ...         HD159176         true
+   0.05069186370709437 -32.58167     9362 ...         HD159176         true
+   0.05069186370709437 -32.58167    10859 ...         HD159176         true
   -0.31119608673831345  6.135278    21939 ...          HD47129         true
   -0.31119608673831345  6.135278    21863 ...          HD47129         true
-  -51.687075256085755  10.68917    26609 ... IRAS F00235+1024         true
-  73.04737400339847  10.18639    25192 ... IRAS F12514+1027         true
-  46.71747372703565 -15.78639    12101 ...      Denis-J1228         true
-  -15.881772371450268 -77.61555    33986 ...         Cha-Ha-3         true
+   -51.687075256085755  10.68917    26609 ... IRAS F00235+1024         true
+     73.04737400339847  10.18639    25192 ... IRAS F12514+1027         true
+     46.71747372703565 -15.78639    12101 ...      Denis-J1228         true
+   -15.881772371450268 -77.61555    33986 ...         Cha-Ha-3         true
 
-This will execute an ADQL query to download the first 10 observations in the XMM-Newton Science Archive. The result of the query will be 
+This will execute an ADQL query to download the first 10 observations in the XMM-Newton Science Archive. The result of the query will be
 stored in the file 'results10.csv'. The result of this query can be printed by doing print(result).
-            
+
 -----------------------------------
 5. Getting table details of XSA TAP
 -----------------------------------
 
-.. code-block:: python
+.. doctest-remote-data::
 
-  >>> from astroquery.esa.xmm_newton import XMMNewton
-  >>>
-  >>> XMMNewton.get_tables()
+  >>> from astroquery.esa.xmm_newton import XMMNewton    # doctest: +IGNORE_OUTPUT
+  >>> XMMNewton.get_tables()  # doctest: +IGNORE_OUTPUT
```

Comment:
It would be actually nice to keep this table listing in the tests. Maybe add a comment line, so if it's failing again we'll know that it's flaky and can ignore it.